src/pydvl/value/sampler.py

```diff
@@ -312,6 +312,26 @@ def weight(cls, n: int, subset_len: int) -> float:
         return float(2 ** (n - 1)) if n > 0 else 1.0
 
 
+class MSRSampler(StochasticSamplerMixin, PowersetSampler[IndexT]):
+    """An iterator to perform sampling of random subsets.
+
+    This sampler does not return any index, it only returns subsets of the data.
+    This sampler is used in :footcite:t:`wang_data_2022`.
```

Comment:
`footcite` doesn't work with mkdocs. Look at [oob.py](https://github.com/aai-institute/pyDVL/blob/develop/src/pydvl/value/oob/oob.py) for an example of how we do it currently.